.vscode/settings.json

```diff
@@ -75,11 +68,19 @@
     "python.analysis.extraPaths": [
         "rerun_py/rerun_sdk"
     ],
-    "ruff.args": [
-        "--config",
-        "rerun_py/pyproject.toml"
+    "ruff.lint.arg": [
+        "--config=rerun_py/pyproject.toml"
     ],
```

Comment:
doesn't exist - but the `ruff.lint.args` below does.

Unless you're on some alpha version where it does?